From af492162772a6ed26fbc8223012348ae4ad000e5 Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Fri, 13 Nov 2020 10:45:59 +0100
Subject: [PATCH] 90multipath: install kpartx's 11-dm-parts.rules

This makes /dev/disk/by-uuid links point to the right device.

(cherry picked from commit 0f89c0dff6b96c9ca09b48a2890399b145f38bdb)

Resolves: #1849021
---
 modules.d/90multipath/module-setup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/90multipath/module-setup.sh b/modules.d/90multipath/module-setup.sh
index a7f250f3..1844a035 100755
--- a/modules.d/90multipath/module-setup.sh
+++ b/modules.d/90multipath/module-setup.sh
@@ -125,6 +125,6 @@ install() {
     inst_rules 40-multipath.rules 56-multipath.rules \
 	62-multipath.rules 65-multipath.rules \
 	66-kpartx.rules 67-kpartx-compat.rules \
-	11-dm-mpath.rules
+	11-dm-mpath.rules 11-dm-parts.rules
 }
 

